libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-wrapper.component.ts

```diff
@@ -191,4 +225,61 @@ export class SkyAgGridWrapperComponent implements AfterContentInit, OnDestroy {
       }
     }
   }
+
+  #updateInlineHelp(column: Column, context?: unknown): void {
+    const colId = column.getId();
+    const columnHeader = this.elementRef.nativeElement.querySelector(
+      `div.ag-header-cell[col-id="${colId}"] div.ag-cell-label-container`
+    ) as HTMLElement;
+    const inlineHelpComponent: Type<unknown> | undefined =
+      column.getColDef().headerComponentParams?.inlineHelpComponent;
+    const currentComponent: ComponentRef<unknown> | undefined =
+      this.#columnInlineHelpComponents.get(colId);
+
+    const getHostElement = (): HTMLElement => {
+      if (!columnHeader.querySelector(`span.sky-control-help-container`)) {
+        const inlineHelpContainer = this.document.createElement('span');
+        inlineHelpContainer.classList.add('sky-control-help-container');
+        columnHeader.insertBefore(inlineHelpContainer, columnHeader.firstChild);
```

Comment:
Create `<span class="sky-control-help-container"></span>` before the header label so the label padding can be adjusted with `.sky-control-help-container + .ag-header-cell-label`.